src/Editor/GUI/Editors/TextureEditor.cpp

```diff
@@ -66,5 +64,5 @@ void TextureEditor::SetVisible(bool visible) {
 void TextureEditor::FileSelected(const std::string& file) {
     std::string destination = Hymn().GetPath() + "/" + texture->path + texture->name + ".png";
     FileSystem::Copy(file.c_str(), destination.c_str());
-    texture->GetTexture()->Load(file.c_str(), texture->srgb);
+    texture->GetTexture()->Load(file.c_str(), false);
```

Comment:
Whoops. false will be implicitly converted to 0 and the wrong Load method called. false needs to be removed from the call.